.travis.yml

```diff
@@ -6,5 +6,5 @@ os:
   - osx
 
 smalltalk:
-  - Squeak-5.2
-  - Squeak-5.1
+  - Squeak64-trunk
+  - Squeak64-5.3
```

Comment:
I think we just remove travis at this point, as it still fails

Reply:
Not quite sure about this, since without travis, coverage is no longer reported.
Travis is actually succeeding, the badge only shows it as failing because it is already targeting the dev branch with the old travis configuration. Does not hurt to keep it, I guess.

Reply:
oh, github action has no coverage report?

Reply:
Seems like it. The smalltalkci readme does only mention coverage for Travis or AppVeyor and the test-ci branch only appearend on coveralls after I reenabled travis.